### PR TITLE
Exclude generated code from jacoco plugin in mysql connector.

### PR DESCRIPTION
**Pull request by Cyril-Engels**
Exclude generated code from jacoco plugin in mysql connector to avoid IncompatibleClassChangeError.

### CODE REVIEW

**Review by C0urante — APPROVED** [review]
LGTM pending a green build

Minor suggestion: we could leave these in as-are for now and skip them by adding `<skip>true</skip>` to the configuration for the plugin, maybe with a small comment above detailing why we're omitting Jacoco for now. Or not, up to you.

**Comment by Cyril-Engels** [comment]
> LGTM pending a green build
> 
> Minor suggestion: we could leave these in as-are for now and skip them by adding `<skip>true</skip>` to the configuration for the plugin, maybe with a small comment above detailing why we're omitting Jacoco for now. Or not, up to you.

@C0urante I added jacoco for the 3 connectors yesterday to get coverage report, as that seems to be a requirement for GA. But build started to fail after that. This change is merely to test if build will still fail with only one connector has jacoco enabled. I probably won't merge it.

**Pull request merged**
